From b18bd67106e366b7dfe39c1fa3eafaef498ac4fe Mon Sep 17 00:00:00 2001
From: Michal Simek <monstr@monstr.eu>
Date: Mon, 8 Oct 2012 13:25:32 +0200
Subject: [PATCH] gpio: Remove ARCH_XILINX reference

Remove this old Kconfig option. It is unused right now.

Signed-off-by: Michal Simek <monstr@monstr.eu>
---
 drivers/gpio/Kconfig |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/gpio/Kconfig b/drivers/gpio/Kconfig
index 6daee40..bd2b68b 100644
--- a/drivers/gpio/Kconfig
+++ b/drivers/gpio/Kconfig
@@ -185,13 +185,13 @@ config GPIO_STA2X11
 
 config GPIO_XILINX
 	bool "Xilinx GPIO support"
-	depends on PPC_OF || MICROBLAZE || ARCH_XILINX || ARCH_ZYNQ
+	depends on PPC_OF || MICROBLAZE || ARCH_ZYNQ
 	help
 	  Say yes here to support the Xilinx AXI/XPS GPIO device
 
 config GPIO_XILINX_PS
 	tristate "Xilinx GPIO PS"
-	depends on ARCH_XILINX || ARCH_ZYNQ
+	depends on ARCH_ZYNQ
 	help
 	  Say yes here to support Xilinx GPIO PS controller
 
-- 
1.7.0.4

